airflow/operators/s3_file_transform_operator.py

```diff
@@ -63,7 +68,8 @@ def __init__(
             self,
             source_s3_key,
             dest_s3_key,
-            transform_script,
+            transform_script=None,
+            select_expression=None,
```

Comment:
This will break the API, can you add the arguments to the end?

Reply:
Sorry if I misunderstand, but I think it doesn't break the API since apply_defaults decorator does not allow positional argument.

Reply:
Ah check